INTITA/Modules/ForgotPassword/ForgotPasswordViewController.swift

```diff
@@ -21,9 +21,6 @@ class ForgotPasswordViewController: UIViewController, Storyboarded {
     
```

Comment:
Is this class should conform Storyboarded? I don't see executing his methods. If not, remove please.

Reply:
It conforms to Storyboarded because we push this view from Coordinator